cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -125,3 +124,33 @@ def test_retire_ansible_service(appliance, catalog_item, request, job_type):
     assert order_request.is_succeeded(method='ui'), msg
     myservice = MyService(appliance, catalog_item.name)
     myservice.retire()
+
+
+@pytest.mark.tier(3)
+@pytest.mark.ignore_stream('upstream')
+def test_check_tower_job_status(appliance, catalog_item, request, config_manager_obj, job_type):
+    """Tests order Ansible Tower catalog item and check status on Jobs page
+    Metadata:
+        test_flag: provision
+
+    Polarion:
+        assignee: nachandr
+        casecomponent: Services
+        caseimportance: medium
+        initialEstimate: 1/4h
+    """
+    template = config_manager_obj.yaml_data['provisioning_data'][job_type]
```

Comment:
I think this needs a wrapper so the test doesn't end with an ERROR in the Call phase because the yaml is missing a job type.

It should skip based on the KeyError that this statement will produce.